tests/helpers/test_helpers.coffee

```diff
@@ -71,6 +71,40 @@ _append = (app, item) ->
     item.appendTo(Ember.$('#ember-testing')[0])
   wait app
 
+_openColorChooser = (app, element = 'body') ->
+  dropdown = find '.color-picker-button .dropdown', element
+  if dropdown.hasClass 'open'
+    wait()
+  else
+    dropdownButton = find '.color-picker-button', element
+    click dropdownButton
+
+_closeColorChooser = (app, element = 'body') ->
+  dropdown = find '.color-picker-button .dropdown', element
+  if dropdown.hasClass 'open'
+    dropdownButton = find '.color-picker-button', element
+    click dropdownButton
+  else
+    wait()
+
+_getSelectedColor = (app) ->
+  active = find '.color-picker-dropdown .active'
+  if active.length
+    rgbVal = active.css('background-color')
+    [_, _, _, r, g, b, a, _] = /(.*?)rgb(a)?\((\d+), (\d+), (\d+)(, (\d+))?\)/.exec(rgbVal)
```

Comment:
I think Peter wrote this originally. But `a` is third (we ignore any alpha channel)